bike.rb

```diff
@@ -0,0 +1,11 @@
+class Bike < Transport
+
+  attr_accessor :max_distance
```

Comment:
accessor provides setter and getter for us. Do we need both of them?

Reply:
used getter , changed code